README.md

```diff
@@ -8,4 +8,17 @@ The predictive performances of the model were evaluated by comparing simulated p
 Schematic illustration of the alveolar and bronchial compartment and subcompartments in the physiologically based biopharmaceutics lung model developed in MoBi.
 
 
-![alt text](https://github.com/Erik-Sjogren/Lung-absorption-model/blob/main/Lung-model.png)
+![alt text](Lung-model.png)
```

Comment:
@Erik-Sjogren always better to refer pictures etc. with their relative names. Then it's safe e.g. to move a repository to another location etc.